index.js

```diff
@@ -1,12 +1,155 @@
-const express = require('express');
-const helmet = require('helmet');
+const express = require("express");
+const helmet = require("helmet");
+const knex = require("knex");
+
+const knexConfig = require("./knexfile");
+
+const db = knex(knexConfig.development);
 
 const server = express();
 
 server.use(express.json());
 server.use(helmet());
 
-// endpoints here
+// ZOOS TABLE ENDPOINTS ***********************
+// GET ENPOINT
+server.get("/zoos", async (req, res) => {
+  try {
+    const zoos = await db("zoos");
+    res.status(200).json(zoos);
+  } catch (error) {
+    res.status(500).json({
+      message: "Houston, we have a problem! Failed to retrieve records."
+    });
+  }
+});
+server.get("/zoos/:id", async (req, res) => {
+  try {
+    const zoo = await db("zoos").where({ id: req.params.id });
+    res.status(200).json(zoo);
+  } catch (error) {
+    res.status(500).json({ message: "we can't get the records" });
+  }
+});
+
+server.post("/zoos", async (req, res) => {
+  try {
+    const newAnimal = req.body;
+    const id = await db("zoos").insert(newAnimal);
+    if (id > 0) {
+      res.status(200).json({ message: "Your record is added" });
+    } else {
```

Comment:
Using status code `201` would better fit the context of creating a new entry in the DB